FFCPayBatchGenerator/Factory/BatchFactory.cs

```diff
@@ -1,24 +1,19 @@
 ﻿
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace FFCPayBatchGenerator.Factory
+namespace FFCPayBatchGenerator.Factory;
```

Comment:
There was a mix of convention between file scoped namespaces and more traditional namespaces declarations.

For consistency I've converted them all to the more modern file scoped namespaces.